### PR TITLE
Linux : Virtual device creation and routing microphone audio via a python script

### PR DESCRIPTION
**Pull request by arushsharma24**
Created a folder 'linux' which contains the following files:- 
1. dynopii.sh : a shell script to create a virtual device named "dynopii" if it does not exist already, and stream microphone audio input into it.
2. a README.md on how to use the script

### CODE REVIEW

**Comment by arushsharma24** [comment]
Update:
Now instead of a shell script, we have a python script (as originally planned for this project) which should fulfill all requirements on a linux system.
Commit : 619e3dd  adds 
1. dynopii.py : A python script that pipes audio from arecord to aplay, and this recorded audio can be further processed if so desired. Also runs the scripts dynopii.sh and close.sh
2. dynopii.sh : Creates the null-sink-module (basically our virtual device) named dynopii and opens pavucontrol.
3. close.sh : Closes the playback, the virtual device and the script. Is run automatically when the python script dynopii.py is quit using Ctrl-C.

Commit f7e9600 updates README accordingly

File: linux/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#
```
Comment (xprilion) [inline linux/README.md]: Remove

File: linux/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## Requirements:-
+    pavucontrol, python module subprocesss, audioop
+#
```
Comment (xprilion) [inline linux/README.md]: Remove

File: linux/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    `$ python3 dynopii.py`
+
+#### Step 2:
+    In the "Playback" tab of the pavucontrol window that opens, and make sure that the playback device for the running "ALSA plug-in[aplay]: ALSA Playback _on_" is 'dynopii'. 
+
+#### Step 3:
+    In your conferencing software, select '_Monitor of dynopii_' as your audio input device.
+
+And your microphone audio is now routed to the conference!
+#
```
Comment (xprilion) [inline linux/README.md:8]: Remove

File: linux/dynopii.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    #this is the part which runs when it is running for the first time
+    echo ""
+    echo ""
+    pacmd load-module module-null-sink sink_name=dynopii_sink sink_properties=device.description=dynopii &
+    pavucontrol
+fi
+
+#arecord -f cd - | python wire.py | aplay -  Will use this when I can get piping working.
+#python hope.py | aplay -f cd       This one should work but I don't know how to work with the subprocess module
+#arecord -f cd - | aplay -
```
Comment (xprilion) [inline linux/dynopii.sh]: Are these lines relevant? If yes, please add proper introduction comments. If not, please remove

**Review by xprilion — CHANGES_REQUESTED** [review]

**Comment by arushsharma24** [comment]
Update : 
Included the code of all the shell scripts into the python file so there is no extra file (in c59265a and 598ddad), everything bundled inside the single dynopii.py file. 
Also removed the superfluous lines as suggested from the readme as well.  (fc2fe7f)

Thank you for reviewing and hope you can give this a look again whenever possible :)

**Review by xprilion — APPROVED** [review]

**Pull request merged**